evap/contributor/templates/contributor_index.html

```diff
@@ -103,6 +106,14 @@
     {% endfor %}
 {% endblock %}
 
+{% block modals %}
+    {{ block.super }}
+    {% trans 'Delegate the preparation' as title %}
+    {% trans 'Do you really want to delegate the preparation of the course <strong id="data-label"></strong>?' as question %}
+    {% trans 'Delegate preparation' as action_text %}
+    {% include 'user_profile_select_modal.html' with modal_id='addEditorModal' title=title form=delegate_selection_form question=question action_text=action_text btn_type='primary' %}
+{% endblock %}
```

Comment:
unless you plan to include that modal in your next PR a second time, i'd say inline all of that. maybe keep the modal_id as a variable, but the rest, i'd inline it.